.pre-commit-config.yaml

```diff
@@ -17,7 +17,7 @@ repos:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
 -   repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: v0.1.13
```

Comment:
this https://github.com/nerfstudio-project/nerfstudio/pull/3369/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711 updates `ruff` version in `pyproject.toml `but not `.pre-commit-config.yaml.`